### ISSUE TITLE
17808: 2016-02-23 18:34:53,473 ERROR backend: g_log purple_account_set_public_alias: assertion 'purple_account_is_connected(account)' failed

### ISSUE THREAD

**Issue by thundergreen** [majn/telegram-purple#245]
17808: 2016-02-23 18:34:53,473 ERROR backend: g_log purple_account_set_public_alias: assertion 'purple_account_is_connected(account)' failed

can someone help me with this issue?

**Comment by majn** [comment]
Do you have a full log?

**Comment by thundergreen** [comment]
Yes here : [See here](https://emevth.no-ip.biz/paste/?7cf2c53cb670ba0c#m/CXZDsOPmbyqyyV6Jbrec6hVZ3ME38fWFbkjNgn40s=)

**Comment by majn** [comment]
Doesnt seem to contain anything fatal except of the warning, does it work?

**Comment by thundergreen** [comment]
Nope. It return internal server error, please reconnect. is it maybe cause of spectrum?

**Comment by thundergreen** [comment]
Is there a possibility to have more debug output?

**Comment by thundergreen** [comment]
in spectrum log i have a backend crash report, but backend log doesn't show anything. That means 1. I need more DEBUG OR Spectrum issue. Dunno. Maybe there's a way to have more DEBUG output?

**Comment by majn** [comment]
You can increase the libtgl logging level in the source code https://github.com/majn/telegram-purple/blob/master/telegram-purple.c#L561 .

Its hard to tell if its a Spectrum issue, I don't know that software at all. It seems to fail an assertion and then restart.

**Comment by thundergreen** [comment]
What do i habe to change before compiling?

Thorsten Fröhlich
Am 23.02.2016 20:41 schrieb "mj" <notifications@github.com>:

**Comment by thundergreen** [comment]
Is tgl automatically installed when I compile and install telegram purple?

Thorsten Fröhlich
Am 23.02.2016 20:55 schrieb "Thorsten Fröhlich" <thundergreen@googlemail.com

**Comment by majn** [comment]
Increase the versbosity in the linked line from 4 to a level like 6 or higher

**Comment by majn** [comment]
> Is tgl automatically installed when I compile and install telegram purple?

no, libtgl is included as a submodule in this project

**Comment by thundergreen** [comment]
Ok will increase ..what's the maximum?

But the submodules will be used ? I mean it seems that it runs ..so it uses
the submodules I guess

Thorsten Fröhlich
Am 23.02.2016 21:00 schrieb "mj" <notifications@github.com>:

**Comment by majn** [comment]
Set it to 6, it means print all DEBUG messages.

**Comment by thundergreen** [comment]
Ok cool thanks will try this

Thorsten Fröhlich
Am 23.02.2016 21:06 schrieb "mj" <notifications@github.com>:

**Comment by silberzwiebel** [comment]
I do have  a similar problem and set the verbosity level to 10. The login via spectrum2 does not work, [here is my purple log](https://gist.github.com/silberzwiebel/71e2ace5c859dbb0d96cc506168a9a76) (I've replaced my actual phone number and domain.).
